debian/changelog

```diff
@@ -0,0 +1,5 @@
+jetson-gpio (2.0.4-1) unstable; urgency=medium
+
+  * Initial release
+
+ -- Andrew Chen <andrewc@nvidia.com>  Tue, 15 Oct 2019 14:18:38 +0800
```

Comment:
We should use a generic email address; see setup.py for the one to use.

Reply:
Actually, it looks like this needs to be by name and email address so that I can sign the package.